tests/benchmarks/cli/stories/use-cases/test_sharing.py

```diff
@@ -15,3 +17,17 @@ def test_sharing(bench_dvc, tmp_dir, dvc, dataset, remote):
     bench_dvc("pull", name="noop")
 
     bench_dvc("checkout", name="noop")
+
+
+@pytest.mark.parametrize("jobs", [None, 1, 2, 4])
+def test_jobs(bench_dvc, tmp_dir, dvc, dataset, remote, jobs):
+    if jobs:
+        args = ("-j", str(jobs))
+
+    bench_dvc("add", dataset)
+    bench_dvc("push", *args)
+
+    shutil.rmtree(dataset)
+    shutil.rmtree(dvc.odb.local.cache_dir)
+
+    bench_dvc("pull", *args)
```

Comment:
This benchmark will take a while to run with big datasets. Maybe unit tests are enough?

Reply:
Yeah, I noticed. I was just trying to run a baseline for another related, not really going to merge this